modules/git/diff.go

```diff
@@ -293,7 +293,7 @@ func GetAffectedFiles(oldCommitID, newCommitID string, env []string, repo *Repos
 	err = NewCommand("diff", "--name-only", oldCommitID, newCommitID).
 		RunInDirTimeoutEnvFullPipelineFunc(env, -1, repo.Path,
 			stdoutWriter, nil, nil,
-			func(ctx context.Context, cancel context.CancelFunc) error {
+			func(_ context.Context, _ context.CancelFunc) error {
```

Comment:
For example, I am not sure whether it is correct to ignore  the `cancel context.CancelFunc`, usually it might be called somewhere.

So maybe we could spend enough time to review the suspicious usages one by one.

Reply:
In this case it's fine. This internal function doesn't need to call cancel or to pay attention to the provided context as if the context is cancelled the stdout pipe should get closed and then the whole thing will stop.